test/framework/functional/message_templates.go

```diff
@@ -49,13 +49,12 @@ var (
 
 func NewApplicationLogTemplate() types.ApplicationLog {
 	return types.ApplicationLog{
-		Timestamp:     time.Time{},
-		Message:       "*",
-		LogType:       "application",
-		ViaqIndexName: "app-write",
-		Level:         "*",
-		Hostname:      "*",
-		ViaqMsgID:     "*",
+		Timestamp: time.Time{},
+		Message:   "*",
+		LogType:   "application",
+		Level:     "*",
```

Comment:
Why we remove `ViaqIndexName`?

Reply:
it is only needed for Elasticsearch so I removed it from ingress and moved it to the ES blocks of configuration.